command/install.go

```diff
@@ -8,57 +8,60 @@ import (
 	"github.com/mitchellh/cli"
 )
 
+//Installer is a interface for Install
 type Installer interface {
 	Install() int
 }
 
+//InstallCommand is a collection
 type InstallCommand struct {
 	// To control this CLI's display
-	Ui cli.Ui
+	UI cli.Ui
 
 	// OS command to execute
 	Cmd *exec.Cmd
 }
 
+//MayaAsNomadInstaller is a collection
 type MayaAsNomadInstaller struct {
 	InstallCommand
 
 	// if the installation is in server mode or client mode
-	is_master bool
+	ismaster bool
 
 	// all maya master ips, in a comma separated format
-	master_ips string
+	masterIps string
 
 	// all maya openebs ips, in a comma separated format
-	client_ips string
+	clientIps string
 
 	// self ip address
-	self_ip string
+	selfIP string
 
 	// the provided master ips in a format understood by Nomad & Consul
-	fmt_master_ips string
+	fmtMasterIps string
 
 	// the provided master ips with rpc ports in a format understood by Nomad
-	fmt_master_ipnports string
+	fmtMasterIpnports string
 
-	// a trimmed version of self_ip
-	self_ip_trim string
+	// a trimmed version of selfIP
+	selfIPTrim string
 
 	// formatted etcd initial cluster
-	etcd_cluster string
+	etcdCluster string
 
 	//contains version info
 	nomad  string
 	consul string
 }
 
-// The public command
+//Install is the public command
 func (c *MayaAsNomadInstaller) Install() int {
 
 	var runop int
 
 	if runop = c.verifyBootstrap(); runop != 0 {
-		//Run the bootstrap only if required
+		//Run the bootstrap only if reqUIred
```

Comment:
n_ reqUIred -> required. 

sounds like a global replace of ui to UI.

Reply:
ohh my bad